frontend/src/routes/Applications/SubscriptionApplication.tsx

```diff
@@ -196,8 +196,8 @@ export function CreateSubscriptionApplication(
     (providerConnection) => providerConnection.metadata?.labels?.['cluster.open-cluster-management.io/type'] === 'ans'
   )
 
-  const hubCluster = useHubCluster(true)
-  const isLocalCluster = !!hubCluster
+  const localHubName = useLocalHubName()
+  const isLocalCluster = !!localHubName
```

Comment:
Here we should be using `useIsHubSelfManaged`... I was thinking `useLocalHubName()` would always return something (defaulting to 'local-cluster'), and the other hook would be used when you need to know for sure if the hub is self-managed. The errors we saw happened when the local hub name is undefined, so I would like to simplify those areas so we are sure there is always a value.